base.sbt

```diff
@@ -0,0 +1,2 @@
+addCommandAlias("gitSnapshots", ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\"")
+enablePlugins(GitBranchPrompt)
```

Comment:
Sure. (Sadly I don't think I can disable it globally in ~/.sbt as it needs to exist to be disabled and I don't want to add sbt-git globally either)